crawler/__init__.py

```diff
@@ -261,16 +314,34 @@ def _debug(cls):
     def info(cls):
         images = cls.__images
         blacklist = cls.__blacklist
-        return {
-            "images": len(images),
-            "images_size": sys.getsizeof(images, 0),
+
+        info = {
             "blacklist": len(blacklist),
             "blacklist_size": sys.getsizeof(blacklist, 0)
         }
 
+        images_count = 0
+        images_size = 0
+        for crawler in images:
+            for site in images[crawler]:
+                site_image_count = len(images[crawler][site])
+                images_count += site_image_count
+                images_size += sys.getsizeof(images[crawler][site])
+                info[crawler + "_" + site] = site_image_count
```

Comment:
could we have an own section for this? so the return could look like the following: 
~~~~
{		
  "images" : 23
  "images_size": 1337 ,
  "blacklist":  42 ,
  "blacklist_size":  9000 , 
  "images_sitespecific" : {
    "myCrawler#site" : 2
  } 
 }
~~~~
so it would be easy to distinguish crawlerspecific stats from the general ones and parse them via any script ...